libs/pika/async_mpi/src/mpi_future.cpp

```diff
@@ -306,6 +305,8 @@ namespace pika { namespace mpi { namespace experimental {
                 // is only used for an assert
                 request_callback_vector[std::size_t(index)].request =
                     MPI_REQUEST_NULL;
+
+                --(detail::get_mpi_info().requests_vector_size_);
```

Comment:
```suggestion
                --(detail::get_mpi_info().active_requests_vector_size_);
```
? as now we don't count the null requests in the vector anymore

Reply:
Renamed the variable.